docs/variants/qemu_q35/building-manual.md

```diff
@@ -46,157 +46,20 @@ Update the submodules in order get latest dependencies.
-
-#### Enabled Dasharo script
-
-```bash
-sed -i 's|//#define\s*IMAGE_SCRIPT.*|#define IMAGE_SCRIPT|' "src/config/general.h"
-```
-
-Get the script:
-
-```bash
-wget https://raw.githubusercontent.com/Dasharo/dasharo-blobs/main/dasharo/dasharo.ipxe
-```
-
-#### Enabled HTTPS
-
-```bash
-sed -i 's|.*DOWNLOAD_PROTO_HTTPS|#define DOWNLOAD_PROTO_HTTPS|g'  "src/config/general.h"
-```
-
-#### Compile
-
-Start the instance of the docker image under the Dasharo/edk2 repository:
-
-```bash
-docker run --rm -it -v $PWD/../edk2:/home/coreboot/edk2 \
-    -v $PWD/../edk2-platforms:/home/coreboot/edk2-platforms \
-    -v $PWD:/home/coreboot/ipxe \
-    -w /home/coreboot/ipxe coreboot/coreboot-sdk:2021-09-23_b0d87f753c \
-    /bin/bash
-```
-
-Export cross compiler prefix:
-
-```bash
-export CROSS_COMPILE="x86_64-elf-"
-```
-
 Compile:
 
 ```bash
-make -C src bin-x86_64-efi-sb/ipxe.efi EMBED=$PWD/dasharo.ipxe BUILD_ID_CMD="echo 0x1234567890" \
-    EXTRA_CFLAGS="-Wno-address-of-packed-member  -m64  -fuse-ld=bfd \
-    -Wl,--build-id=none -fno-delete-null-pointer-checks -Wlogical-op -march=nocona \
-    -malign-data=abi -mcmodel=large -mno-red-zone -fno-pic"
+./build.sh qemu
 ```
 
-#### Build EDK II
-
-Change directory:
-
-```bash
-cd ../edk2
-```
-
-Setup the environment variables with the following command
-
-```bash
-source edksetup.sh
-```
-
-Compile EDK II base tools:
-
-```bash
-make -C BaseTools
-```
-
-Update the PACKAGES_PATH variable:
-
-```bash
-export EDK2_PLATFORMS_PATH="$HOME/edk2-platforms"
-export PACKAGES_PATH="$WORKSPACE:\
-$HOME/ipxe/src/bin-x86_64-efi-sb/:\
-$EDK2_PLATFORMS_PATH/Platform/Intel:\
-$EDK2_PLATFORMS_PATH/Silicon/Intel:\
-$EDK2_PLATFORMS_PATH/Features/Intel:\
-$EDK2_PLATFORMS_PATH/Features/Intel/Debugging:\
-$EDK2_PLATFORMS_PATH/Features/Intel/Network:\
-$EDK2_PLATFORMS_PATH/Features/Intel/OutOfBandManagement:\
-$EDK2_PLATFORMS_PATH/Features/Intel/PowerManagement:\
-$EDK2_PLATFORMS_PATH/Features/Intel/SystemInformation:\
-$EDK2_PLATFORMS_PATH/Features/Intel/UserInterface"
-```
-
-To build the image simply invoke the following command
-
-```bash
-build -a IA32 -a X64 -t GCC5 -b RELEASE -p OvmfPkg/OvmfPkgX64.dsc
-```
-
-Once the build is completed you should see output as follows:
-
-```text
-GUID cross reference file can be found at /home/coreboot/edk2/Build/OvmfX64/RELEASE_GCC5/FV/Guid.xref
-
-FV Space Information
-SECFV [5%Full] 212992 total, 11568 used, 201424 free
-PEIFV [15%Full] 917504 total, 139640 used, 777864 free
-DXEFV [46%Full] 12582912 total, 5849680 used, 6733232 free
-FVMAIN_COMPACT [41%Full] 3440640 total, 1430512 used, 2010128 free
-
-- Done -
-  Build end time: 21:52:36, Oct.22 2023
-  Build total time: 00:01:41
-```
-
-The Dasharo firmware image can be found below given path:
-
-```bash
-/home/coreboot/edk2/Build/OvmfX64/RELEASE_GCC5/FV/OVMF_CODE.fd
-/home/coreboot/edk2/Build/OvmfX64/RELEASE_GCC5/FV/OVMF_VARS.fd
-```
+The resulting coreboot image will be placed in the coreboot directory as
```

Comment:
If the command is `./build.sh` then "the coreboot directory" is the current directory and we might as well call it that way for clarity.
```suggestion
The resulting coreboot image will be placed in the current directory as
```

Reply:
https://github.com/Dasharo/docs/compare/2a47bdd5f3eae81d35cc9545010489747bac76c6..fd1fc06fa12f4d9dde6a66b01a567657c6bb1a5d